sdk/samples/iot/README.md

```diff
@@ -55,7 +55,7 @@ To run the samples, ensure you have the following programs and tools installed o
 
 - Have the most recent version of [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases) installed (more instructions can be found [here](https://docs.microsoft.com/azure/iot-pnp/howto-use-iot-explorer)) and connected to your Azure IoT Hub if running a Plug and Play sample:
 
-  *Executables:* `paho_iot_hub_pnp_sample`, `paho_iot_hub_pnp_component_sample`
+  *Executables:* `paho_iot_hub_pnp_sample`, `paho_iot_pnp_component_sample`
 
```

Comment:
will `paho_iot_hub_pnp_sample` also get updated at somepoint?  Why the difference between it and `paho_iot_pnp_component_sample`?

Reply:
Yup tried to give reasoning above in the pr comment but yes, once the sample is updated to use the pnp api

Reply:
you did. that's just me not reading. smh.